package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "learnr",
+  "version": "0.10.0",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/rstudio/learnr/issues"
+  },
+  "homepage": "https://pkgs.rstudio.com/learnr",
+  "main": "",
+  "engines": {
+    "node": ">= 14"
+  },
+  "scripts": {
+    "build": "npm run build:tutorial-js && npm run build:tutorial-i18n && npm run build:tutorial-format",
+    "build:tutorial-js": "babel src-js/tutorial -d inst/lib/tutorial",
+    "build:tutorial-i18n": "babel src-js/i18n -d inst/lib/i18n",
+    "build:tutorial-format": "babel src-js/format -d inst/rmarkdown/templates/tutorial/resources",
+    "lint": "standard --fix src-js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rstudio/learnr.git"
+  },
```

If this isn't being published, then it is fair to set `private: true` and remove other parts of the config.
```suggestion
  "name": "learnr",
  "private": true,
  "engines": {
    "node": ">= 14"
  },
  "scripts": {
    "build": "npm run build:tutorial-js && npm run build:tutorial-i18n && npm run build:tutorial-format",
    "build:tutorial-js": "babel src-js/tutorial -d inst/lib/tutorial",
    "build:tutorial-i18n": "babel src-js/i18n -d inst/lib/i18n",
    "build:tutorial-format": "babel src-js/format -d inst/rmarkdown/templates/tutorial/resources",
    "lint": "standard --fix src-js"
  },
```